ocaml/src/Makefile

```diff
@@ -46,7 +47,7 @@ DIROPT = -I `ocamlfind query newspeak` -I fixpoint -I `ocamlfind query ocamlgrap
 
 
 CAMLC   =ocamlc $(DIROPT)
-CAMLOPT =ocamlopt -g -w Ael -warn-error +a-7-3-52-50-42 $(DIROPT)
+CAMLOPT =ocamlopt -g -w Ael  -bin-annot $(DIROPT)
```

Comment:
Yes, but stricter compiler flags  are probably desirable.
I installed zarith with opam and I get 
Warning 58: no cmx file was found in path for module Z, and its interface was not compiled with -opaque

Removing Warning 58 as error would be sufficient.

PS: I indeed forgot to include the relevant code...